### ISSUE TITLE
Numeric value with variable radix position - numericInput: true

### ISSUE THREAD

**Issue by douglasdtc** [RobinHerbots/Inputmask#1319]
I used a JSFiddle as an example: http://jsfiddle.net/msj5qfu1/159/

When the user types **12345**, it should format as **123.45**, and it works, but, when the user types **123.5** it should format exactly as **123.5** or **123.50** (the 0 doesn't make a difference).

I've tried with all the options I could find and none of them seem to work.

Is this something that needs implementation or am I doing it wrong? Thank you.

**Comment by douglasdtc** [comment]
@RobinHerbots , there's another thing. When I delete the value, it stays as "0,00". It should be "" (null).  Is there any option to enforce this? Nullable didn't do the job and the placeholder as "0,00", "0.00", "0" and "" also didn't work. Thank you.

**Comment by RobinHerbots** [comment]
@douglasdtc , 

Why do you need the numericInput?  Anyway, this needs to be implemented for numericInput: true.

**Comment by douglasdtc** [comment]
We use numericInput so that the user could start typing from rtl, as the majority of fields are monetary values.

**Comment by RobinHerbots** [comment]
@douglasdtc , 

Did you already tried with numericInput: false?

Anyway , I will add the functionality to numerinput true;  Can you also create a separate issue for the 0,00 deleting.

**Comment by RobinHerbots** [comment]
@douglasdtc , 

Just some thoughts.
NumericInput true
Say I type 123    => result on screen 1.23
then I type . => result 123.<caret>00
I guess you would expect that typing 45 will result in 123.45
But the behavior of numericInput is shifting to the left resulting in 12345.00

The numericInput false will do exactly.  Except the input start at the integer part
start => 0.00
type 123  => 123<caret>.00
type . => 123.<caret>00
type 45 => 123.45

**Comment by douglasdtc** [comment]
@RobinHerbots 
Yeah, indeed the behavior you describe is the correct behavior, and the most usual.
This is actually an enhancement, as it will allow the user to have the default behavior (current one), and also, give him the option to force the position of the radix point.

**Comment by douglasdtc** [comment]
Hello @RobinHerbots , any updates about this issue? Thank you.

**Comment by douglasdtc** [comment]
Is this implemented in 4.0 @RobinHerbots ?

**Comment by RobinHerbots** [comment]
@douglasdtc , 

No it isn't.  Sorry for that.  Already a year later, ...

Anyway, what about

> Just some thoughts.
> NumericInput true
> Say I type 123 => result on screen 1.23
> then I type . => result 123.00
>> I guess you would expect that typing 45 will result in 123.45
>> But the behavior of numericInput is shifting to the left resulting in 12345.00
